.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
```

Comment:
this should ideally also make sure it doesn't run on forks

Reply:
This won't run on forks unless they push a tag to it with this format. To prevent it from ever running we could do:

```diff
 jobs:
   release:
+    if: github.repository == 'yannickcr/eslint-plugin-react'
     runs-on: ubuntu-latest
```

Reply:
Forks have all tags by default, and I often push tags to my fork first to ensure tests pass before i push it to the main repo. That addition sounds great :-) (unless there's something that can say "if this is not a fork" without hardcoding the repo name)